modules/nested-accordion/assets/js/frontend/handlers/nested-accordion.js

```diff
@@ -34,4 +36,74 @@ export default class NestedAccordion extends Base {
 			$items[ index ].appendChild( element );
 		} );
 	}
+
+	applyDefaultStateCondition() {
+		if ( ! this.elements ) {
+			return;
+		}
+
+		const accordionItems = this.elements.$items,
+			defaultState = this.getDefaultStateCondition(),
+			stateFirstExpanded = 'first_expended';
+
+		if ( stateFirstExpanded === defaultState ) {
+			accordionItems[ 0 ].setAttribute( 'open', '' );
+		} else {
+			accordionItems.each( ( _, item ) => item.removeAttribute( 'open' ) );
+		}
+	}
+
+	getDefaultStateCondition() {
+		return elementorFrontend.utils.controls.getControlValue( this.getElementSettings(), 'default_state', '' );
```

Comment:
Do we need a function in order to get the control value?
cant we use something like:
`const { default_state: defaultState } = this.getElementSettings();`
in the desired location?
I see we only use it once.